app/models/service_change_request.rb

```diff
@@ -0,0 +1,4 @@
+class ServiceChangeRequest < ApplicationRecord
+  belongs_to :service
```

Comment:
Was thinking, this needs to be a nullable field, as proposed service additions and proposed service changes should really be stored in the same table, otherwise the approval process will get quite complex

Reply:
optional: true?

Reply:
Yep :+1: will add that